### DISCUSSION TITLE
Add new report types: Vendor retail analytics and Brand Analytics

### DISCUSSION THREAD

**Discussion by Guerri114** [saleweaver/python-amazon-sp-api#263] [Feature requests]
Amazon launched additional report types... can these be added to python-amazon-sp-api?

- GET_BRAND_ANALYTICS_SEARCH_TERMS_REPORT
- GET_VENDOR_SALES_DIAGNOSTIC_REPORT
- GET_VENDOR_INVENTORY_HEALTH_AND_PLANNING_REPORT
- GET_VENDOR_DEMAND_FORECAST_REPORT

**Comment by saleweaver** [comment]
https://github.com/saleweaver/python-amazon-sp-api/commit/9f3b455afbd199f8e1fc73fa1042d8ae712beed7

**Reply by saleweaver** [reply]
Added them
